presto-cassandra/src/test/java/io/prestosql/plugin/cassandra/CassandraServer.java

```diff
@@ -70,18 +68,16 @@ public static synchronized void start()
 
         log.info("Starting cassandra...");
 
-        System.setProperty("cassandra.config", "file:" + prepareCassandraYaml());
-        System.setProperty("cassandra-foreground", "true");
-        System.setProperty("cassandra.native.epoll.enabled", "false");
```

Comment:
Was `cassandra-foreground` and `cassandra.native.epoll.enabled` something important?

Reply:
Yes, we couldn't run tests without these properties on EmbeddedCassandra.